src/components/LogoText/index.js

```diff
@@ -19,7 +19,7 @@ const LogoText = ({ children, width, linkTo, alt, srcLight, srcDark }) => {
           />
         </a>
       </div>
-      <div className={clsx(styles.sideImageR)}><ReactMarkdown>{children}</ReactMarkdown></div>
+      <div className={clsx(styles.sideImageR)}>{children.props.children}</div>
```

Comment:
That removes the markdown component. `ReactMarkdown` so i guess that was the issue?

Reply:
Yeah, @weichweich, I think something changed, and it wasn't needed anymore. Instead, the value passed to the component was already a React component and could be rendered without anything else.